storage_backend_ftp/components/ftp_adapter.py

```diff
@@ -0,0 +1,163 @@
+# Copyright 2021 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```

Comment:
the license of the module is LGPL. Can you check all licenses in this module pls?